migrations/versions/ae68621248af_primary_venue.py

```diff
@@ -0,0 +1,58 @@
+"""Primary venue
+
+Revision ID: ae68621248af
+Revises: 2441cb4f44d4
+Create Date: 2018-11-23 01:52:58.790889
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'ae68621248af'
+down_revision = '2441cb4f44d4'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.create_table('project_venue_primary',
+        sa.Column('project_id', sa.Integer(), nullable=False),
+        sa.Column('venue_id', sa.Integer(), nullable=False),
+        sa.Column('created_at', sa.DateTime(), nullable=False),
+        sa.Column('updated_at', sa.DateTime(), nullable=False),
+        sa.ForeignKeyConstraint(['project_id'], ['project.id'], ondelete='CASCADE'),
+        sa.ForeignKeyConstraint(['venue_id'], ['venue.id'], ondelete='CASCADE'),
+        sa.PrimaryKeyConstraint('project_id')
+        )
+    op.execute(sa.DDL('''
+        CREATE FUNCTION project_venue_primary_validate() RETURNS TRIGGER AS $$
```

Comment:
Is it possible to add a comment somewhere here to explain what this function does?

Reply:
This is part of `add_primary_relationship`. It's from Coaster.